Blackjack/Score.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+
+namespace Blackjack
+{
+    public class Score
+    {
+        public int Calculate(Hand hand)
+        {
+            var sum = 0;
+            for (var i = 0; i < hand.Cards.Count; i++)
+            {
+                var rank = (int)hand.Cards[i].Rank;
+                if (rank > 10) sum += 10;
+                if (rank > 1 && rank <= 10) sum += rank;
+                if (rank == 1) sum += 11;

```

Comment:
what if there's more than 1 ace or if adding the ace will cause the hand to be > 21? You mentioned today that the aces should be calculated last after all the other cards in the hand have been calculated - I think that's a good path to follow :)